tests/admin/test-class-admin-recommended-replace-vars.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * WPSEO plugin test file.
+ *
+ * @package WPSEO\Tests\Admin
+ */
+
+/**
+ * Unit Test Class.
+ */
+class WPSEO_Admin_Recommended_Replace_Vars_Test extends WPSEO_UnitTestCase {
+
+	/** @var WPSEO_Admin_Recommended_Replace_Vars */
+	protected $class_instance;
+
+	/**
+	 * Set up the class which will be tested.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->class_instance = new WPSEO_Admin_Recommended_Replace_Vars();
+	}
+
+	/**
+	 * Test that determine_for_term can detect a category.
```

Comment:
Can you changes `Test` to `Tests`. This is the form we are using in PHP documentation.

Reply:
This is applicable for mostly of the tests in this class